Queryable/Queryable/View/SearchResultsView.swift

```diff
@@ -64,7 +64,7 @@ struct SearchResultsView: View {
         case .HAS_RESULT:
             // Has result
             VStack {
-                if photoSearcher.totalUnIndexedPhotosNum > 0 {
+                if photoSearcher.totalUnIndexedPhotosNum > 0 || PHPhotoLibrary.authorizationStatus(for: .readWrite) == .limited {
```

Comment:
Always show the button if the library has a limited authorization status, since it's always true that there may be more photos the user could index.

Reply:
Reasonable.